### ISSUE TITLE
csdp now available as a Sage package

### ISSUE THREAD

**Issue by dimpase** [emil79/flagmatic#3]
Nowadays, one can (assuming the Sage was built from source, so that one can install packages needing a C compiler) just do

   sage: install_package('csdp')

This will in particular install csdp executables and libraries into $SAGE/local, and so they should be accessible for flagmatic.

**Comment by kcrisman** [comment]
I agree that at least the documentation should be updated for this!  See also http://stackoverflow.com/questions/27108300/problems-installing-flagmatic
